conda-forge.yml

```diff
@@ -1,6 +1,8 @@
+azure:
+  store_build_artifacts: true
```

Comment:
@xylar see here.  The bot just alpha sorted the keys

Reply:
Oh, I totally missed that.  Okay.